base/gocb_utils.go

```diff
@@ -114,7 +103,11 @@ func GoCBCoreTLSRootCAProvider(caCertPath string) (func() *x509.CertPool, error)
 	}, nil
 }
 
-func getRootCAs(caCertPath string) (*x509.CertPool, error) {
+func getRootCAs(CACertUnsetTlsSkipVerify bool, caCertPath string) (*x509.CertPool, error) {
```

Comment:
I think it's better to leave this function with a single purpose - retrieve the rootCAs - and have callers make the decision about whether to call it in the first place based on TlsSkipVerify.